pkgs/top-level/all-packages.nix

```diff
@@ -20749,6 +20749,15 @@ in
 
   syncthing-tray = callPackage ../applications/misc/syncthing-tray { };
 
+  syncthingtray-minimal = libsForQt5.callPackage ../applications/misc/syncthingtray { };
+  syncthingtray = libsForQt5.callPackage ../applications/misc/syncthingtray {
```

Comment:
I'd expect `synthingtray` to refer to the default configuration of `synthingtray`. This may be confusing if a user looks at the expression and wonders why the flags are set differently. Better to set the appropriate defaults in the expression and override them in  `synthingtray-minimal` (whose name indicates that it deviates from the default configuration).

Reply:
You are right.